docs/admin/system-information.rst

```diff
@@ -1574,49 +1574,161 @@ Tables need to be recreated
    cluster check is failing. Follow the instructions below to get this cluster
    check passing.
 
-This check warns you if there are tables that need to be recreated for
-compatibility with future major versions of CrateDB.
+This check warns you if your cluster contains tables that you need to reindex
+before you can upgrade to a future major version of CrateDB.
 
-If you try to upgrade to the next major version of CrateDB with tables that
-have not been recreated, CrateDB will refuse to start.
+If you try to upgrade to a later major CrateDB version without reindexing the
+tables, CrateDB will refuse to start.
 
-To recreate a table, you have to create new tables, copy over the data and
-rename or remove the old table.
+CrateDB Table Version Compatibility Scheme
```

Comment:
```suggestion
CrateDB table version compatibility scheme
```

Reply:
we switched to sentence case for all headings last year (it is considered better style. see parameter one here: https://practicaltypography.com/headings.html)